build/configs/cy4390x/board_metadata.txt

```diff
@@ -0,0 +1,26 @@
+###########################################################################
```

Comment:
minor suggestion, could you please change : "to set a name of kernel binary" to "to set the name of kernel binary"